argoverse/evaluation/stereo/utils.py

```diff
@@ -0,0 +1,252 @@
+# <Copyright 2021, Argo AI, LLC. Released under the MIT license.>
+"""Stereo utilities to support the Argoverse stereo evaluation."""
+
+from pathlib import Path
+from typing import List
+
+import cv2
+import disparity_interpolation
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+from argoverse.evaluation.stereo.constants import (
+    DEFAULT_ABS_ERROR_THRESHOLDS,
+    DEFAULT_REL_ERROR_THRESHOLDS,
+    DISPARITY_NORMALIZATION,
+    LOG_COLORMAP,
+    NUM_EVAL_ERROR_REGIONS,
+    NUM_EVAL_PIXEL_REGIONS,
+)
+
+
+def compute_disparity_error(
+    pred_fpath: Path,
+    gt_fpath: Path,
+    gt_obj_fpath: Path,
+    figs_fpath: Path,
+    abs_error_thresholds: List[int] = DEFAULT_ABS_ERROR_THRESHOLDS,
+    rel_error_thresholds: List[float] = DEFAULT_REL_ERROR_THRESHOLDS,
+    save_disparity_error_image: bool = False,
+) -> pd.DataFrame:
+    """
+    Compute the disparity errors.
```

Comment:
should go on the same line above
```
) -> pd.DataFrame:
    """Compute the disparity errors.

        Args:
```

https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings